src/api/routes/v2/blocks.ts

```diff
@@ -174,5 +181,53 @@ export const BlockRoutesV2: FastifyPluginAsync<
     }
   );
 
+  fastify.get(
+    '/:height_or_hash/signer-signature',
```

Comment:
I think plural might make more sense here given that it's a paginated endpoint i.e. `signer-signatures`

Reply:
There's weirdness around this because IIUC the initial (and eventual) design from stacks-core is a single `signer_signature` field for a block, and this array value is only an epoch3.0 quirk. So singular naming I think would be for future-compatibility (it would return an array with 1 item, still kinda weird).

... but if core does make that change in the future, we'd probably still want to receive and expose this list of all signer signatures, so plural naming is also future-compatible. I'll make that change.